Library/Homebrew/cmd/audit.rb

```diff
@@ -610,7 +610,7 @@ def audit_text
       problem "\"Formula.factory(name)\" is deprecated in favor of \"Formula[name]\""
     end
 
-    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin]
+    if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin] && text !~ %r[kibana]
```

Comment:
Maybe
```ruby
 if text =~ /system "npm", "install"/ && text !~ %r[opt_libexec\}/npm/bin] && formula.name != "kibana"
```
Or something. Just in case the word `kibana` suddenly becomes commonplace or anything.